examples/RNOneSignal/index.android.js

```diff
@@ -0,0 +1,297 @@
+/**
```

Comment:
Is this a new example in this PR? Or something that got added by mistake

Reply:
I just used the official react native initialization, and merged the changes.
This is makes a lot of sense since bugs could be easily tested on a fresh environment.